From: Michal Hocko <mhocko@suse.com>
Subject: xen: kabi: x86/speculation/l1tf: Increase l1tf memory limit for Nehalem+
Patch-mainline: Never, SUSE-Xen specific
References: bnc#1105536

hide new x86_cache_bits from the kabi checker. Nobody should be touching
this part of the structure.

Signed-off-by: Michal Hocko <mhocko@suse.com>

Automatically created from "patches.kabi/0001-x86-kabi-speculation-l1tf-Increase-l1tf-memory-limit-for-.patch" by xen-port-patches.py

--- a/arch/x86/include/mach-xen/asm/processor.h
+++ b/arch/x86/include/mach-xen/asm/processor.h
@@ -124,11 +124,13 @@ struct cpuinfo_x86 {
 #ifndef CONFIG_XEN
 	u32			microcode;
 #endif
+#ifndef __GENKSYMS__
 	/*
 	 * Address space bits used by the cache internally
 	 * NOTE: only to be used for l1tf mitigation
 	 */
 	u8			x86_cache_bits;
+#endif
 } __attribute__((__aligned__(SMP_CACHE_BYTES)));
 
 #define X86_VENDOR_INTEL	0
